package.json

```diff
@@ -21,12 +21,13 @@
   },
   "dependencies": {
     "common": "0.2.x",
-    "lodash": "3.10.x",
-    "request": "2.64.x"
+    "lodash": "^4.17.10",
+    "pug": "^2.0.3",
```

Comment:
You don't need this line: `pug` is a dependency of `mocha`, so the update to `mocha ^5.2.0` below takes care of the `npm install` warning about jade/pug.

Reply:
fixed